components/tinaMarkdownComponents/docAndBlogComponents.tsx

```diff
@@ -330,9 +349,23 @@ function FormatHeaders({ children, level }) {
     typeof window !== 'undefined' ? window.location.pathname : '';
   const linkHref = `${currentUrl}#${id}`;
 
+  const styles = {
+    1: 'text-orange-600 text-4xl font-bold mt-4 mb-4',
+    2: 'text-orange-500 text-3xl font-semibold mt-4 mb-3',
+    3: 'text-blue-900 text-xl font-medium mt-2 mb-2',
+    4: 'text-orange-500 text-xl font-medium mt-2 mb-2',
+    5: 'text-orange-500 text-lg font-medium mt-2 mb-1',
+    6: 'text-gray-500 text-base font-normal mt-2 mb-1',
+  };
+
   return (
-    <HeadingTag id={id} className="relative cursor-pointer">
-      <a href={linkHref} className="no-underline group">
+    <HeadingTag id={id} className={`${styles[level]} relative cursor-pointer`}>
+      <a
+        href={linkHref}
+        className="no-underline group"
+        onClick={() => navigator.clipboard.writeText(`tina.io${linkHref}`)}
+      >
+        {' '}
         {children}
         <FiLink className="ml-2 opacity-0 group-hover:opacity-100 transition-opacity duration-200 inline-flex mb-2" />
       </a>
```

Comment:
look into why the table stying has changed - maybe this was impacted when we removed some of the styled.jsx css stuff

Reply:
https://github.com/tinacms/tina.io/issues/2583